src/Event/VendorEvent.php

```diff
@@ -42,6 +43,17 @@ public function __construct(string $type, array $identifiers, string $identifier
         $this->vendorId = $vendorId;
     }
 
+    /**
+     * Change the current state.
+     *
+     * @param $newVendorState
```

Comment:
missing type...

Reply:
I'm not sure what the type should be as they come from const, which have not type?

Reply:
Fixed